### ISSUE TITLE
Production Deployment

### ISSUE THREAD

**Issue by SPlanzer** [linz/gazetteer#201]
#### Task
* Run https://github.com/linz/gazetteer/blob/master/src/sql/gazetteer_geometry_views.sql
*  update database version code
```
UPDATE gazetteer.system_code
SET value='2.0.0'
WHERE value='1.8.2'
```
* #205 is a dependency and must be addressed

#### Definition of Done
Once all steps above complete.

**Comment by SPlanzer** [comment]
# Assessment of readiness for prd deployment  

## UAT
Prd plugin release cannot happen until UAT is complete. 

As I understand it, UAT is completed with the issues in the below section having been raised. @CMKMS-LINZ can you confirm this is the case? 

### UAT issues open. 
* #204
  * This is has been fixed and is awaiting UAT signoff 
*  #206
  * This functionality has long been problematic and is affected by database source code rather than the QGIS code. I suggest we place this on the backlog as not to stall the release of the plugin to QGIS3 and make a plan to address it. If this must be addressed before go live, it will significantly stall the plugin being released to QGIS3. 
* #207
  * Same as above - This is not the plugin code but the database code responsible for this functionality. I recommended putting this on the backlog and making a plan to address it when resources are available. 

## Next
@CMKMS-LINZ can you confirm whether you are happy to release to production with #204 Resolved but #206 and #207 outstanding?

If so, I will perform the release to the LINZ production plugin repository and ask that you support NZGB users migrate to QGIS3

**Comment by CMKMS-LINZ** [comment]
I haven't even started UAT due to my BAU commitments and other project work I've been doing. Theoretically other members in the team are also expected to do UAT but there will be a hurdle of getting them up and running and telling them what to do.

The five issues raised were just the initial showstopper, then four things encountered immediately after opening it for the first time.

**Comment by CMKMS-LINZ** [comment]
Jill says 'not until next week' - I'll endeavour to complete UAT by following my own test plans tomorrow.

**Comment by billgeo** [comment]
No longer relevant
